### ISSUE TITLE
Adds support of "+" and "-" between xsd:dateTime and xsd:duration to SPARQL

### ISSUE THREAD

**Issue by Tpt** [eclipse/rdf4j#185]
It would be nice to add support of xsd:dateTime +/- xsd:duration and xsd:dateTime - xsd:dateTime (returning an xsd:duration) operations to SPARQL.

Jena already supports such operators. see https://github.com/apache/jena/blob/master/jena-arq/src/main/java/org/apache/jena/sparql/expr/nodevalue/NodeValueOps.java

Probably blocked by #164

**Comment by barthanssens** [comment]
Seems related to (or duplicate of ?) of #524 ?
